skills/csharp/experimental/itsmskill/Dialogs/SkillDialogBase.cs

```diff
@@ -51,7 +52,15 @@ public class SkillDialogBase : ComponentDialog
                 throw new Exception("You must configure an authentication connection before using this component.");
             }
 
-            AddDialog(new MultiProviderAuthDialog(Settings.OAuthConnections));
+            AppCredentials oauthCredentials = null;
+            if (Settings.OAuthCredentials != null &&
+                !string.IsNullOrWhiteSpace(Settings.OAuthCredentials.MicrosoftAppId) &&
```

Comment:
Empty string?

Reply:
white space is better :) it includes check for empty string